validator/src/bootstrap.rs

```diff
@@ -682,21 +682,25 @@ mod without_incremental_snapshots {
             let rpc_peers = rpc_peers.unwrap();
             blacklist_timeout = Instant::now();
 
+            let mut filtered_rpc_peers = vec![];
+            for rpc_peer in rpc_peers.iter() {
+                if !bootstrap_config.only_known_rpc
+                    && is_known_validator(&rpc_peer.id, &validator_config.known_validators)
```

Comment:
```suggestion
                    || is_known_validator(&rpc_peer.id, &validator_config.known_validators)
```

I think this will fix the CI stable failure

Reply:
Thank you @t-nelson ! Fixed :)